### DISCUSSION TITLE
Are Pods going to be implemented with Quadlet?

### DISCUSSION THREAD

**Discussion by sigulete** [containers/podman#20880] [General]
I have been using `podman generate` to create **systemd units** to handle my **pods** with no issues. But, recently I have received a message advising that it is deprecated in favour of Quadlet.

The problem is that Quadlet is not fully replacing `podman generate`, because it handles only the creation of **systemd units** for **Containters**.

According to the documentation [podman-systemd.unit](https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html), it should be possible to create `.pod` files and a `[POD]` section is described in this manual as well.

Is this going to be implemented in Podman in the near future?

**Comment by shyzus** [comment]
#20762 from what I can tell the basics are there. However note that the version of the docs you were looking at is "latest". This is not always the same as the version you may be using. The last version released is 4.8.0. Seems the changes in the pull request didn't make it into the last release.

**Reply by rhatdan** [reply]
Yes this was just merged and will be in podman 5.0 early next year.

**Comment by sigulete** [comment]
Ok, thanks.
